modules/postgresql/versions.tf

```diff
@@ -19,6 +19,5 @@ terraform {
   required_providers {
     google = "~> 3.5"
     null   = "~> 2.1"
-    random = "~> 2.2"
```

Comment:
You should at least pin a minimum version.

Reply:
Could you please review the latest commit?
Thanks.